patches/putcols.c.patch

```diff
@@ -1,19 +0,0 @@
---- cfitsio3470/putcols.c	2018-03-01 10:28:51.000000000 -0600
-+++ cfitsio3470/putcols.c	2018-12-14 08:39:20.000000000 -0600
-@@ -157,7 +157,16 @@
-
-          for (;jj < twidth; jj++)    /* fill field with blanks, if needed */
-          {
--           *buffer = ' ';
-+             if ( fits_use_standard_strings() ) {
-+                 if (snull[0] == ASCII_NULL_UNDEFINED) {
-+                     *buffer = ' ';
-+                 } else {
-+                     *buffer = '\0';
-+                 }
-+             } else {
-+                 *buffer = ' ';
-+             }
-+
```

Comment:
This patch is needed for string handling in python 3. We cannot remove it.

Reply:
unless the upstream file has been patched, which as far as I know it has not been.

Reply:
It doesn't apply to 3470.  Should it be adapted to it then?

Reply:
The line number was off by one.  Fixed now and re-added.